plat/mediatek/common/mtk_sip_svc.c

```diff
@@ -56,11 +72,9 @@ static uint64_t mediatek_sip_handler(uint32_t smc_fid,
 		SMC_RET1(handle, ret);
 
 	default:
-		ERROR("%s: unhandled SMC (0x%x)\n", __func__, smc_fid);
-		break;
+		return mediatek_plat_sip_handler(smc_fid, x1, x2, x3, x4,
+			cookie, handle, flags);
```

Comment:
We will do more code sharing with other platform team. We move common functions  to common folder and define these functions as weak functions. If the same functions need to be customized by platform, re-implement these weak functions.